### PR TITLE
CASMINST-6498 add cani procedures for adding hardware

### PR DESCRIPTION
**Pull request by jacobsalmela**
# Description

- Add `cani` procedures to add cabinets and blades.
- Link to procedures as an alternative path in existing documentation

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by shunr-hpe — APPROVED** [review]

File: operations/cani/Add_A_Cabinet_To_SLS.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ncn# cani alpha add cabinet -L # see supported cabinet types
+ncn# cani alpha add cabinet <cabinet type> --auto # add a cabinet using recommended values
+```
+
+[Apply](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_apply/) the changes, writing the new data directly to SLS.
+
+```text
+ncn# cani alpha session apply # push changes to SLS
+```
+
```
Comment (rsjostrand-hpe) [inline operations/cani/Add_A_Cabinet_To_SLS.md:24]: May want a pointer that the cabinet will not have any blades in it, and blades will need to be added.

File: operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Add A Blade To A Cabinet In SLS Using CANI
+
+Begin by [initializing a session](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_init/), which will import data from SLS and create metadata in SLS' `ExtraProperties` field allowing the process to continue.
+
+```text
+ncn# cani alpha session init csm \
```
Comment (rsjostrand-hpe) [inline operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md]: Need to adhere to: https://github.com/Cray-HPE/docs-csm/blob/release/1.6/introduction/documentation_conventions.md#command-prompt-conventions

File: operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+```text
+ncn# cani alpha add blade -L # see supported blade types
+ncn# cani alpha add blade <blade type> --auto # add a blade using recommended values
+```
+
+[Apply](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_apply/) the changes, writing the new data directly to SLS.
+
+```text
+ncn# cani alpha session apply # push changes to SLS
+```
```
Comment (rsjostrand-hpe) [inline operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md]: We most likely need to to copy over the non-sls steps into this doc/refractor the common steps/link to the steps in https://github.com/Cray-HPE/docs-csm/blob/0638f927cf5ac02b6e5880a38c78d8ca96b144fe/operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md as we are only modifying SLS, and not touching the other services.

File: operations/cani/Add_A_Cabinet_To_SLS.md (hunk 1 above)
Comment (jacobsalmela) [reply operations/cani/Add_A_Cabinet_To_SLS.md:24]: Fixed

File: operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md (hunk 2 above)
Comment (jacobsalmela) [reply operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md]: Fixed

File: operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md (hunk 3 above)
Comment (jacobsalmela) [reply operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md]: No SLS tasks are actually in this doc, but I added the cani steps as a prerequisite

**Review by rsjostrand-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**